backend/eventloop/db_reconciler_test.go

```diff
@@ -547,7 +557,7 @@ var _ = Describe("DB Reconciler Test", func() {
 				repoCredentialCr = managedgitopsv1alpha1.GitOpsDeploymentRepositoryCredential{
 					ObjectMeta: metav1.ObjectMeta{
 						Name:      "test-repo-" + string(uuid.NewUUID()),
-						Namespace: "test-k8s-namespace",
+						Namespace: "argocd",
```

Comment:
Is it necessary to create this in `argocd`? I think it should instead be created in either the old value, or in the namespace defined in `apiNamespace`.

Reply:
`argocd` is required because we're using 
```
_, _, _, gitopsEngineInstance, _, err := db.CreateSampleData(dbq)
Expect(err).To(BeNil())
```
this will create a gitopsEngineInstance in `argocd` namepsace, I have tried to alter the ns to some other ns for eg. the default ns `gitops-service-argocd` but it clashes with the `argoCDNamespace` creation during fake client setup. So went with `argocd` which isn't used elsewhere to get good validity.